client/src/pages/guide/english/html/elements/blockquote-tag/index.md

```diff
@@ -3,13 +3,22 @@ title: Blockquote Tag
 ---
 ## Blockquote Tag
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/elements/blockquote-tag/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+### Purpose
+The HTML <blockquote> element shows that the enclosed text is an extended quotation. Usually, this is rendered visually by indentation.  
```

Comment:
These two sentences are directly from the MDN article you linked to. Can you please re-word.